format.md

```diff
@@ -25,65 +34,64 @@ The version specifies the version of the certificate payload encoding, and the s
         * LE
 
 ## Verification
-To verify a doughnut requires two steps be carried out:
+To verify a doughnut:
 
 1. Verify the issuer's signature
     1. Separate `<VERSION><PAYLOAD>` from `<SIGNATURE>`
     2. Extract the issuer public key from `PAYLOAD` as `ISSUER`
         * How this is done depends on the payload version
     3. Verify `SIGNATURE` is valid given a message of `<VERSION><PAYLOAD>` and a signer of `ISSUER`
         * How this is done depends on the signing method
-2. Verify the payload
+2. Verify the user's account is the `HOLDER`
```

Comment:
This looks great

The only update I'd question is:

```
2. Verify the user's account is the `HOLDER`
```

`holder` is used in a number of ways. Verification isn't always necessary, or is carried out against a signature of something other than the Doughnut. Future payload versions may have support for `holders` that aren't cryptographic (possibly things like email addresses). While this is simpler, there are also no holder, and holder list variants.

Will be complex trying to cover every verification case in spec. At this point I'd recommend leaving the holder verification to the client.

Everything else is great, and improves the spec clarity.

Reply:
Done